lwallet/Main.hs

```diff
@@ -145,20 +145,21 @@ addTxSubmit mvar = modifySharedAtomic mvar (\(TxCount submitted failed sending)
 addTxFailed :: Mockable SharedAtomic m => SharedAtomicT m TxCount -> m ()
 addTxFailed mvar = modifySharedAtomic mvar (\(TxCount submitted failed sending) -> return (TxCount submitted (failed + 1) sending, ()))
 
-runCmd :: forall ssc ctx m. MonadWallet ssc ctx m => SendActions m -> Command -> CmdCtx -> m ()
+runCmd :: SendActions LightWalletMode -> Command -> CmdCtx -> LightWalletMode ()
 runCmd _ (Balance addr) _ =
     getBalance addr >>=
     putText . sformat ("Current balance: "%coinF)
 runCmd sendActions (Send idx outputs) CmdCtx{na} = do
     skeys <- getSecretKeys
-    etx <-
-        withSafeSigner (skeys !! idx) (pure emptyPassphrase) $ \mss ->
-        runEitherT $ do
-            ss <- mss `whenNothing` throwError "Invalid passphrase"
-            lift $ submitTx
-                (immediateConcurrentConversations sendActions na)
-                ss
-                (map (flip TxOutAux []) outputs)
+    let skey = skeys !! idx
+        curAddr = makePubKeyAddress $ encToPublic skey
```

Comment:
Ah, here we use root key as address directly, so nevermind.
Unless heavy and light wallets share the same key storage - then everything is sad, and it is for a long time already